demo/api/src/main.ts

```diff
@@ -39,6 +41,19 @@ async function bootstrap(): Promise<void> {
         }),
     );
 
+    app.use(
+        helmet({
+            contentSecurityPolicy: {
+                // disable all defaults (https://www.npmjs.com/package/helmet/v/4.6.0#:~:text=If%20no%20directives%20are%20supplied%2C%20the%20following%20policy%20is%20set)
+                useDefaults: false,
+                directives: {
+                    "default-src": helmet.contentSecurityPolicy.dangerouslyDisableDefaultSrc,
+                    // locally: allow localhost in frame-ancestors to enable including files from the API in iframes in admin
+                    "frame-ancestors": `'self' ${process.env.NODE_ENV === "development" ? config.adminUrl : ""}`,
+                },
```

Comment:
This has the disadvantage that it's not defined in the library. So it doesn't work out-of-the-box in all our projects